pandas/tests/groupby/test_groupby.py

```diff
@@ -2846,3 +2846,12 @@ def test_sum_of_booleans(n):
     result = df.groupby("groupby_col").sum()
     expected = DataFrame({"bool": [n]}, index=Index([1], name="groupby_col"))
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("method, n", [("head", 1), ("nth", 0)])
```

Comment:
Can you add first, last and tail as well?

Reply:
Done. And this time?